apps/services/routers.py

```diff
@@ -46,7 +47,7 @@
             )
         }
     },
-    tags=["Services application"],
+    tags=["Student dashboard"],
```

Comment:
pls double check all schemas, bcs i thnk i merged release version into ur branch not correct